src/main/java/controller/MainController.java

```diff
@@ -129,4 +138,27 @@ private void convertExcelRows() {
 		});
 	}
 
+	/**
+	 * Verify the code quality based on the Rules created and sends the results to
+	 * be displayed in the QualityRulesResultFrame
+	 */
+	private void checkCodeQualityAndShow() {
+		String[][] results = getCodeQualityResults();
+		// TODO get real column names
```

Comment:
Should we create a ticket for the TODO's? I think it could be important to keep easier track of them - just small tasks so we won't forget them